packages/core/integration-tests/test/integration/pug-include-extends/index.pug

```diff
@@ -2,3 +2,4 @@ extends ./base.pug
 
 block content
   include other.pug
+  include /absolute/nested.pug
```

Comment:
interesting. how does pug resolve that? is it absolute to the filesystem or some other dir?

Reply:
- `basedir: string`
> The root directory of all absolute inclusion.